config/initializers/rollbar.rb

```diff
@@ -28,7 +28,8 @@
   # Valid levels: 'critical', 'error', 'warning', 'info', 'debug', 'ignore'
   # 'ignore' will cause the exception to not be reported at all.
   # config.exception_level_filters.merge!('MyCriticalException' => 'critical')
-  config.exception_level_filters.merge!('ActionController::RoutingError' => 'ignore')
+  config.exception_level_filters.merge!('ActionController::RoutingError' => 'ignore',
+                                        'Stoplight::Error::RedLight' => 'ignore')
```

Comment:
since we're not going to send red lights to rollbar (good idea) can we log an Action when a stoplight is triggered? would it be better to show the stoplights in the caesar UI?